test/v3/controllers/RetrieveBalanceAndTransactionsControllerSpec.scala

```diff
@@ -50,16 +53,31 @@ class RetrieveBalanceAndTransactionsControllerSpec
 
         runOkTest(expectedStatus = OK, maybeExpectedResponseBody = Some(mtdResponseJson))
       }
+
+      "the request is valid when isPOARelevantAmount switch is enabled" in new Test {
```

Comment:
Can we move this to a more specific test of `adjustFields` (see comment on controller) on the model case class?